config/src/main/java/com/networknt/config/Config.java

```diff
@@ -424,7 +422,7 @@ private InputStream getConfigStream(String configFilename, String path) {
                 logger.info("Unable to load config " + Encode.forJava(configFilename) + ". Looking for the same file name with extension yaml...");
             } else if (configFilename.endsWith(CONFIG_EXT_YAML)) {
                 logger.info("Unable to load config " + Encode.forJava(configFilename) + ". Looking for the same file name with extension json...");
-            } else {
+            } else if (configFilename.endsWith(CONFIG_EXT_JSON)) {
                 System.out.println("Unable to load config '" + Encode.forJava(configFilename.substring(0, configFilename.indexOf("."))) + "' with extension yml, yaml and json from external config, application config and module config. Please ignore this message if you are sure that your application is not using this config file.");
             }
```

Comment:
Should the error message change along with this condition change?

Reply:
I think it's good, `else if (configFilename.endsWith(CONFIG_EXT_JSON))` equals to `else` in this case, because currently only three types of extensions.
the error message means we cannot find any config with these extensions.
The place to log this is weird, but it does its functionality.